### DISCUSSION TITLE
v1.5.2

### DISCUSSION THREAD

**Discussion by n-elie** [metgem/metgem#121] [General]
This version corrects a bug in export metadata where filter and sorting are ignored

**Comment by pvantwer** [comment]
Hello n-elie,
Downloading the last version on my windows machine :

Édition	Windows 10 Professionnel
Version	22H2
Installé le	‎22/‎11/‎2020
Build du système d’exploitation	19045.5011
Expérience	Windows Feature Experience Pack 1000.19060.1000.0

I receive this error message:

Traceback (most recent call last):
  File "MetGem-script.py", line 2, in <module>
  File "metgem\gui.py", line 100, in run
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "pyarrow\__init__.py", line 65, in <module>
ImportError: DLL load failed while importing lib: La procédure spécifiée est introuvable.

Any tricks to fix this problem?

Many thanks in advance

**Reply by n-elie** [reply]
Hi pvantwer,

There is likely some conflict with files left from the old version. You will have to uninstall MetGem, make sure the installation directory is empty and install the new version again.
This should fix your problem.

**Reply by pvantwer** [reply]
Thank you n-elie for you rapid answer!

For the record, I manually supressed the Metgem Folder on the C: disk, then the shortcut, and finally the metgem icon in the uninstall interface of windows.
After this I install the new version and it works indeed!

pvantwer

**Reply by n-elie** [reply]
Nice to read that. Feel free to ask you have any other question

**Comment by Yuan-666-lab** [comment]
Hello n-elie，
Downloading the last version on my windows machine :
Édition Windows 11 Professionnel
When I use metgem to perform UMAP analysis, a pop-up window of unhandledexpection appears and the analysis cannot be performed. How should I solve this problem?
![image](https://github.com/user-attachments/assets/7c7f5a4c-8df7-4cdc-9bfc-9392c2e9f3c6)
